doc/classes/PhysicsPointQueryParameters2D.xml

```diff
@@ -22,7 +22,7 @@
 			The physics layers the query will detect (as a bitmask). By default, all collision layers are detected. See [url=$DOCS_URL/tutorials/physics/physics_introduction.html#collision-layers-and-masks]Collision layers and masks[/url] in the documentation for more information.
 		</member>
 		<member name="exclude" type="RID[]" setter="set_exclude" getter="get_exclude" default="[]">
-			The list of objects or object [RID]s that will be excluded from collisions.
+			The list of object [RID]s that will be excluded from collisions. See e.g. [method CollisionObject2D.get_rid].
```

Comment:
```suggestion
			The list of object [RID]s that will be excluded from collisions. See also [method CollisionObject2D.get_rid].
```
Or just "see". Applies to all of them.

Reply:
I went with `e.g.` because it's not the only way to get an `RID`. People who don't know about `RID`s will most likely just need the `get_rid` method on collision objects, but there is also [`body_create`](https://docs.godotengine.org/en/latest/classes/class_physicsserver3d.html#class-physicsserver3d-method-body-create), [`get_collider_rid`](https://docs.godotengine.org/en/latest/classes/class_kinematiccollision3d.html#class-kinematiccollision3d-method-get-collider-rid), etc. Writing `also` would make the text more vague in my opinion.

Reply:
I've added an explanation of what to do in the most common case.